crates/ide-completion/src/tests/use_tree.rs

```diff
@@ -273,9 +271,39 @@ pub use $0;
             md bar
             kw crate::
             kw self::
+        "#]],
+    );
+}
+
+#[test]
+fn pub_suggest_use_tree_super_acc_to_depth_in_tree() {
```

Comment:
Let's add a third case here which should also show `super::`
```rs
mod foo {
    use $0;
}
```

Reply:
Added this test, alongwith one more for not suggesting `super::` when a non `super` kw has appeared in the segment chain